src/GeneExpressionTSnePlot.js

```diff
@@ -157,17 +122,25 @@ const GeneExpressionScatterPlot = (props) => {
         align: `center`,
         symbolHeight: 5,
         symbolWidth: 450
+      },
+    plotOptions: {
```

Comment:
Good spot! This component used to have plot options, but it looks like a was a little overzealous last time I modified the tooltip, and left behind some invalid Highcharts options (see line [91](https://github.com/ebi-gene-expression-group/scxa-experiment-page-tsne-plot-view/pull/22/files#diff-f73afcc84481662a0e5b85cdbcdff830R91)). Could you remove them so we don't have invalid options lying around?

Reply:
Yes. This `plotOptions` I added is to make all points same style after gene expression json payload is clustered into multiple series as well. I cannot access the line 91 by the link. Where is the invalid Highcharts options you would like to remove?